frontend/src/components/dashboard/DashboardHeader.jsx

```diff
@@ -23,6 +23,9 @@ export default class DashboardHeader extends React.Component {
     const blockPromise = new BlocksApi()
       .getBlockInfo(searchValue)
       .catch(() => {});
+    const blockHeightPromise = new BlocksApi()
+      .searchBlocks(searchValue)
+      .catch(() => {});
```

Comment:
It will search by a height *prefix* (i.e. not an exact search). I suggest we just use `getBlockInfo(blockId)`, where `blockId` is going to be either block hash or block height and then change the query from `WHERE blocks.hash = :hash` to `WHERE blocks.hash = :blockId OR blocks.height = :blockId`. Once we change the implementation of `getBlockInfo`, we don't need this separate call, as `blockPromise` will already search by hash or height.

Reply:
got it